app/controllers/volunteers_controller.rb

```diff
@@ -222,8 +222,8 @@ def knight
     end
 
     volunteer = Volunteer.find(params[:volunteer_id])
```

Comment:
I left this find here in case we wanted to apply the pattern [here](https://github.com/boulder-food-rescue/food-rescue-robot/blob/master/app/controllers/assignments_controller.rb#L20-L23)

If not, I'll pull this down into Knight.